services/src/main/java/org/keycloak/authorization/util/Tokens.java

```diff
@@ -60,11 +57,10 @@ public static AccessToken getAccessToken(String accessToken, KeycloakSession key
         return null;
     }
 
-    public static boolean verifySignature(KeycloakSession keycloakSession, RealmModel realm, String token) {
+    public static boolean verifySignature(KeycloakSession keycloakSession, String token) {
         try {
             JWSInput jws = new JWSInput(token);
-            PublicKey publicKey = keycloakSession.keys().getRsaPublicKey(realm, jws.getHeader().getKeyId());
-            return RSAProvider.verify(jws, publicKey);
+            return keycloakSession.jwt().verify(jws);
```

Comment:
Isn't this another bolierplate code we can also remove by just calling:

```java
keycloakSession.jwt().verify(token);
```

Reply:
Yah. I changed it to TokenManager and now it gets a String input and the expected return type

Much better now :)